src/components/TimeEstimator/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file Time estimator.
+ * @copyright IBM Security 2019
+ */
+
+import Time16 from '@carbon/icons-react/lib/time/16';
+
+import classnames from 'classnames';
+import { node, string } from 'prop-types';
+import React from 'react';
+
+import { getComponentNamespace } from '../../globals/namespace';
+
+const namespace = getComponentNamespace('time-estimator');
+
+/**
+ * Time estimators display the estimated amount of time it takes to complete a flow.
+ */
+const TimeEstimator = ({ children, className, ...props }) => (
```

Comment:
`...props` is different from our consistent use of `...other` and Carbon's consistent use of `...rest` -- could we use one of those two instead? 😅

Reply:
Good catch - I thought I changed this!